### PR TITLE
replicated map stats added to mancenter service

### PR DESCRIPTION
**Pull request by bilalyasar**
this is just initial work and prototype.

### CODE REVIEW

**Comment by emrahkocaman** [comment]
Seems fine so far

**Comment by jerrinot** [comment]
@bilalyasar Are you still working on this one? The last commits is almost 1 month old.

**Comment by mesutcelik** [comment]
@bilalyasar ,
What is the status of this PR?

**Comment by bilalyasar** [comment]
@mesutcelik, @jerrinot i am going to finalise this PR and mancenter side PR today.

**Comment by bilalyasar** [comment]
updated.

**Comment by bilalyasar** [comment]
verify

**Comment by bilalyasar** [comment]
@emrahkocaman, @muratayan Can you review this PR? If it is ok, we can merge this PR.

**Comment by muratayan** [comment]
verify

File: hazelcast/src/main/java/com/hazelcast/monitor/impl/MemberStateImpl.java [hunk 1]
```diff
@@ -289,6 +315,7 @@ public String toString() {
                 + ", runtimeProps=" + runtimeProps
                 + ", mapStats=" + mapStats
                 + ", multiMapStats=" + multiMapStats
+                + ". replicatedMapStats=" + replicatedMapStats
```
Comment (muratayan) [inline hazelcast/src/main/java/com/hazelcast/monitor/impl/MemberStateImpl.java]: I think it should be a comma?

**Comment by muratayan** [comment]
:+1:

File: hazelcast/src/main/java/com/hazelcast/monitor/impl/MemberStateImpl.java (hunk 1 above)
Comment (bilalyasar) [inline hazelcast/src/main/java/com/hazelcast/monitor/impl/MemberStateImpl.java]: yes, thanks.

**Comment by bilalyasar** [comment]
updated PR, also added new tests.

**Comment by mesutcelik** [comment]
:+1:

**Comment by mesutcelik** [comment]
@emrahkocaman ,
Can you have a look one last time?

**Comment by emrahkocaman** [comment]
:+1:

**Pull request merged**